mobile_app_connector/controllers/json_request.py

```diff
@@ -46,7 +54,7 @@ def __init__(self, *args):
                 self.jsonrequest = {}
                 # PUT The GET parameters as the parameters for the controller
                 self.params = {
-                    key: val for key, val in self.httprequest.args.iteritems()
+                    key: val for key, val in self.httprequest.values.iteritems()
```

Comment:
Are you sure we don't lose any parameters by changing this? Shouldn't we merge the two fields?

Reply:
https://werkzeug.palletsprojects.com/en/0.15.x/wrappers/#werkzeug.wrappers.BaseRequest.values